SafeguardSessions/modules/error/VersionErrors.pqm

```diff
@@ -0,0 +1,17 @@
+let
+    ErrorBase = Extension.ImportModule("ErrorBase.pqm"),
+    SupportedVersions.Versions = Extension.ImportFunction("Versions", "SupportedVersions.pqm"),
+    VersionErrors.NotSupportedVersion = (version as text) =>
+        ErrorBase(
+            "Not Supported Version",
+            "The version of SPS you tried to connect to is not supported.",

```

Comment:
As we have talked about it, we should write the connector version and the sps version in the error message as well.

Reasoning:

Since the version is not included in the connector file name (which is actually not a problem because the user can rename it after downloading), I believe that after six months of downloading and using the connector, it is guaranteed that they will forget which connector version they downloaded. Therefore, I wouldn't rely on finding out the connector's supported SPS from the documentation.

The ultimate solution to the customer's problem, with a 99% chance, would be to download a new connector. To do this, they would need to navigate to the release page of the connector. So, this is what I would do:
![image](https://github.com/OneIdentity/SafeguardPowerBI/assets/121802692/6c22865d-fc14-4ae8-86ac-017d79efcbc0)

Reply:
Done

Reply:
Validated by TC: Your version of the connector (#[ConnectorVersion]) is not compatible with your SPS version (#[SPSVersion]). For a connector version that is compatible with your SPS version, visit the official release page of the connector: ...